lib/ui/LandingPage.dart

```diff
@@ -0,0 +1,194 @@
+import 'dart:ui';
+
+import 'package:farmsmart_flutter/ui/common/ActionSheet.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:farmsmart_flutter/ui/mockData/MockActionSheetViewModel.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class _Constants {
+  static final EdgeInsets generalPadding =
+      EdgeInsets.only(left: 32, right: 32, top: 40, bottom: 18);
+  static final double bottomContainerHeight = 350;
+  static final int imageFlexValue = 2;
+  static final double subtitleLineSpace = 10;
+  static final double actionLineSpace = 34;
+  static final EdgeInsets actionTapArea = EdgeInsets.all(10);
+}
+
+class LandingPageViewModel {
+  String detailText;
+  String actionText;
+  String footerText;
+
+  String headerImage;
+  String subtitleImage;
+  Function confirmAction;
```

Comment:
what do we confirm? --- would "continueAction" be better maybe?

Reply:
Perfect !